src/repositories/wirelessProfiles.ts

```diff
@@ -0,0 +1,131 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2019
+ * SPDX-License-Identifier: Apache-2.0
+ * Author : Ramu Bachala
```

Comment:
did he really write this? :) I've been removing author actually  as you get line by line traceability as to whose touched the code with git.